src/sql/workbench/api/common/extHostModelViewDialog.ts

```diff
@@ -653,9 +671,17 @@ export class ExtHostModelViewDialog implements ExtHostModelViewDialogShape {
 		}
 		this.updateButton(dialog.okButton);
 		this.updateButton(dialog.cancelButton);
+		let dialogWidth: DialogWidth = 'narrow';
```

Comment:
Here you have the default be narrow but everywhere else it seems to be wide - why the difference?

Reply:
not really, Dialog and Wizard has different default values, for dialogs, we used to allow authors to specify isWide option, and the default is false, the new default value is 'narrow', for wizards, we only support wide dialog, so the new default value is 'wide'.